x/crosschain/keeper/keeper_cross_chain_tx.go

```diff
@@ -51,6 +47,9 @@ func (k Keeper) SetCctxAndNonceToCctxAndInTxHashToCctx(ctx sdk.Context, send typ
 			Tss:       tss.TssPubkey,
 		})
 	}
+	if send.CctxStatus.Status == types.CctxStatus_Aborted {
+		k.AddZetaAccounting(ctx, send.GetCurrentOutTxParam().Amount)
```

Comment:
1. The function name AddZetaAccounting can be more specific. 
2. Do you need to test whether the outbound is supposed to be ZETA token transfer, not some other tokens?

Reply:
Thanks , good catch !